common/environment/setup/install.sh

```diff
@@ -274,3 +274,18 @@ _vcompletion() {
 			;;
 	esac
 }
+
+_vicons() {
+	local icon_name_scheme="$1" name="$2"
+
+	if [ -z "$icon_name_scheme" ]; then
+		msg_red "$pkgver: vicons: 1 argument expected: <icon_name_scheme>\n"
+		return 1
+	fi
+
+	_name="${name:-${pkgname}.png}"
+	for x in 16 32 64 128 256 512 1024; do
```

Comment:
no comment on whether I'd want to merge this, but don't hardcode the sizes, not all pkgs that do this use all these sizes

Reply:
Yes, that's why we're checking if the file exists. But you're right, it would look way cleaner (and better for the performance) if the resolutions can be passed as a requires argument.

Reply:
Iterating available sizes makes function more useful. For now this is missing some popular sizes and scalables.